src/app/repositories/[...repoid]/page.tsx

```
@@ -3,7 +3,7 @@ import { Metadata } from 'next'
 import { notFound } from 'next/navigation'
 import Content from './Content'
 import Loading from 'src/components/Loading/Loading'
-import { fetchRepositoryMetadata } from 'src/data/queries/repositoryQuery'
+import { fetchRepository, fetchRepositoryMetadataGQL } from 'src/data/queries/repositoryQuery'
```

ESLint

[error] 'fetchRepositoryMetadataGQL' is defined but never used.

(no-unused-vars)